frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -671,3 +693,17 @@ export function getValuesMode(
 
   return "none";
 }
+
+/**
+ *
+ * @param {import("metabase-types/types/Field").Field} field
+ * @param {import("metabase-types/types/Parameter").Parameter} parameter
```

Comment:
Types are wrong. `field` is coming from `metabase-lib` (it's a class), and parameter is `UiParameter` from `metabase-lib/parameters/types`. But I'd remove this comment because it will become outdated very shortly.

Reply:
I was using the same type from `isNumberParameter`'s function signature

https://github.com/metabase/metabase/blob/2bd929c6fc1e7a0d0077c7cee903f9dd2ad817da/frontend/src/metabase-lib/parameters/utils/parameter-type.ts#L35

which seems to receive the same `parameter` we have here.
https://github.com/metabase/metabase/blob/2bd929c6fc1e7a0d0077c7cee903f9dd2ad817da/frontend/src/metabase/parameters/components/ParameterValueWidget.jsx#L234

We seem to have types in a lot of places, so that's why I was confused.

But agree on removing this comment since it'll be refactored soon.